docs/development/deployment.md

```diff
@@ -60,6 +61,204 @@ If you want to reset your environment you can run the `hack/util-update-app-of-a
+upstream    https://github.com/redhat-appstudio/infra-deployments.git (fetch)
+upstream    https://github.com/redhat-appstudio/infra-deployments.git (push)
+```
+
+```shell
+export MY_GIT_FORK_REMOTE=origin
+```
+
+#### GitHub setup
+
+Create a new organization, see the [GitHub docs: Creating a new organization from scratch][github-create-org].
+This is the organization where you will later install the Pipelines as Code application. When testing
+the PaC integration features of StoneSoup, you will need to use repositories from this organization
+(or fork existing repositories into this org).
+
+Copy the organization name to `preview.env`. For example:
+
+```shell
+export MY_GITHUB_ORG=<gh_username>-stonesoup
+```
+
+Next, you will need an access token. See the [GitHub docs: Creating a personal access token][github-get-access-token].
+The token needs all the `repo` permissions and the `delete_repo` permission.
+
+Copy the token value to `preview.env`:
+
+```shell
+export MY_GITHUB_TOKEN=ghp_***
+```
+
+#### Quay setup
+
+Create a new [Quay.io](https://quay.io) organization. If you need a new email address for the organization,
+you can use the `+` trick - e.g. `<gmail_username>+quayorg@gmail.com`.
+
+Copy the organization name to `preview.env`. For example:
+
+```shell
+export IMAGE_CONTROLLER_QUAY_ORG=<quay_username>-stonesoup
+```
+
+Next, you need to generate an access token for your Quay.io organization. See the instructions for
+`IMAGE_CONTROLLER_QUAY_TOKEN` in [preview.env][preview-template.env]. Once you have created the token,
+copy its value to `preview.env`:
+
+```shell
+export IMAGE_CONTROLLER_QUAY_TOKEN=***
+```
+
+#### PaC setup
+
+Follow the [Pipelines as Code: Setup Manually][pac-setup-manual] instructions to create the GitHub application.
+You can create the application in your organization settings (<https://github.com/organizations/MY_STONESOUP_ORG/settings/apps>)
+or your personal settings (<https://github.com/settings/apps>). If you create the app under your organization,
+you can allow it `Only on this account`, otherwise you will have to allow `Any account`.
+
+Compared to the Pipelines as Code documentation, you will want to set these fields differently:
+
+* **GitHub Application Name**: use a unique name, e.g. `<gh_username>-stonesoup PaC`
+* **Webhook URL**: use a dummy url, e.g. <https://example.org>. The `preview.sh` script will fix the
+  webhook url automatically.
+* **Webhook secret**: leave it empty, the `preview.sh` script will set the secret automatically.
+
+Once you have created the GitHub app, copy its App ID to `preview.env`:
+
+```shell
+export PAC_GITHUB_APP_ID=<number>
+```
+
+Generate a private key for the application (in the GitHub UI; refer to the Pipelines as Code docs linked
+above). Encode the private key as base64 and copy it to `preview.env`:
+
+```shell
+base64 --wrap=0 <path to downloaded key>
+```
+
+```shell
+export PAC_GITHUB_APP_PRIVATE_KEY=<output of the command above>
+```
```

Comment:
```suggestion
above). Paste the base64 encoded private key to `preview.env`:
```shell
export PAC_GITHUB_APP_PRIVATE_KEY=$(base64 < /PATH/TO/YOUR/DOWNLOADED/PRIVATE_KEY.pem)
```

something like this? since it's easier

Reply:
My workflow was copy the base64 encoded key into preview.env and then delete it. Though I guess there's no real reason not to keep the key around :thinking:

Reply:
That's a good point :+1: